src/Pact/Types/Command.hs

```diff
@@ -93,20 +94,23 @@ import Pact.Types.Scheme (PPKScheme(..), defPPKScheme)
 data Command a = Command
   { _cmdPayload :: !a
   , _cmdSigs :: ![UserSig]
+  , _cmdSessionPubKey :: Maybe PublicKeyText
```

Comment:
Makes sense, thanks for the explanation! The PR is updated now so that `Command` and `Payload` do not include a session public-key. So session-pubkey isn't exposed via any API of pact nor any of the downstream projects that use pact, but a hypothetical new container service would be able to set it while provisioning its `EvalEnv`. For example a hypothetical new pact server than manages user authentication and sessions could set up session pubkey. (As we discussed on slack. restating it here for the github record).